pkg/virt-launcher/virtwrap/converter/archconverter.go

```diff
@@ -0,0 +1,39 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+package converter
+
+import (
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+type archConverter interface {
+	addGraphicsDevice(vmi *v1.VirtualMachineInstance, domain *api.Domain, c *ConverterContext)
+	isUSBNeeded(c *ConverterContext, vmi *v1.VirtualMachineInstance) bool
```

Comment:
I have dropped it from `isUSBNeeded`. I have left it in `addGraphicsDevice`, since even though only amd64 needs a single boolean from it, giving the function a pointer is just as cheap as copying the boolean.

Reply:
Fair enough, thanks